ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -421,6 +421,88 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-serial,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
if this is expected to fail  let's set always_run:false

Reply:
this will make it manual trigger

Reply:
Right, see https://github.com/openshift/release/pull/14756#discussion_r555354155